lib/standards/aria-roles.js

```diff
@@ -674,6 +684,12 @@ const ariaRoles = {
     nameFromContent: true,
     childrenPresentational: true
   },
+  suggestion: {
+    type: 'landmark',
+    requiredAttrs: ['insertion', 'deletion'],
```

Comment:
This is [required owned roles not attributes](https://w3c.github.io/aria/#suggestion)

```suggestion
    requiredOwned: ['insertion', 'deletion'],
```